web/looking-good/src/pages/user-profile-edit.js

```diff
@@ -1,32 +1,50 @@
-import Backdrop from '@material-ui/core/Backdrop'
-import CircularProgress from '@material-ui/core/CircularProgress'
-import FormErrors from 'src/components/form-errors'
-import MuiAlert from '@material-ui/lab/Alert'
-import React, { useEffect, useState } from 'react'
-import Snackbar from '@material-ui/core/Snackbar'
-import useAuth from 'src/hooks/use-auth'
 import {
+  Avatar,
+  Backdrop,
   Button,
-  Card,
-  CardActions,
-  CardContent,
+  CircularProgress,
   Container,
   FormControl,
   Grid,
+  Snackbar,
   TextField,
-  Typography,
 } from '@material-ui/core'
+import makeStyles from '@material-ui/core/styles/makeStyles'
+import MuiAlert from '@material-ui/lab/Alert'
 import { Formik } from 'formik'
-import { makeStyles } from '@material-ui/core/styles'
 import { useRouter } from 'next/router'
+import React, { useEffect, useState } from 'react'
+import FormErrors from '../components/form-errors'
+import useAuth from '../hooks/use-auth'
```

Comment:
We have webpack setup to allow absolute import paths, so we should be importing from `src/components/form-errors`

Reply:
Cool, I love that! My IDE put the relative paths in for me, will fix :)